packages/amplify-category-function/src/provider-utils/awscloudformation/utils/packageFunction.ts

```diff
@@ -0,0 +1,36 @@
+// For legacy purposes, the method builds and packages the resource
+
+import { pathManager } from 'amplify-cli-core';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import { Packager } from '../types/packaging-types';
+import { getRuntimeManager } from './functionPluginLoader';
+
+/**
+ * Packages lambda source code and artifacts into a lambda-compatible .zip file
+ */
+export const packageFunction: Packager = async (context, resource) => {
```

Comment:
nit: add types for parameters

Reply:
TS infers the types from the Packager interface